keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/VersionComparators.kt

```diff
@@ -37,9 +38,9 @@ class TagComparator(
   private val semverComparator = NullSafeComparator(SEMVER_COMPARATOR, false)
   private val increasingComparator = NullSafeComparator(INCREASING_COMPARATOR, false)
 
```

Comment:
feels like all the naming changes in this file are not necessary

Reply:
You're right. I was just bugging me that similar implementations used different names for these variables so I changed them. Please indulge me this time? 🙂